src/test/java/dk/alexandra/fresco/suite/spdz/TestSpdzComparison.java

```diff
@@ -135,4 +137,27 @@ public void test_compareEQ_Sequential() throws Exception {
 		runTest(new ComparisonTests.TestCompareEQ(),
 				EvaluationStrategy.SEQUENTIAL, StorageStrategy.IN_MEMORY);
 	}
+	
+	@Test
+	public void test_isSorted() throws Exception {
+		runTest(new SortingTests.TestIsSorted(),
+				EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
+	}
+	
+	@Test
+	public void test_compareAndSwap() throws Exception {
+		runTest(new SortingTests.TestCompareAndSwap(),
+				EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
+	}
+	@Test
+	public void test_Sort() throws Exception {
```

Comment:
Will do.
Regards
Morten V. Christiansen
Fra: Kasper Damgård <notifications@github.com<mailto:notifications@github.com>>
Svar til: aicis/fresco <reply@reply.github.com<mailto:reply@reply.github.com>>
Dato: onsdag den 24. februar 2016 kl. 23.27
Til: aicis/fresco <fresco@noreply.github.com<mailto:fresco@noreply.github.com>>
Cc: "Morten V. Christiansen" <morten.v.christiansen@alexandra.dk<mailto:morten.v.christiansen@alexandra.dk>>
Emne: Re: [fresco] Sorting (#18)


In src/test/java/dk/alexandra/fresco/suite/spdz/TestSpdzComparison.java<https://github.com/aicis/fresco/pull/18#discussion_r54017987>:

> @@ -135,4 +137,27 @@ public void test_compareEQ_Sequential() throws Exception {
>               runTest(new ComparisonTests.TestCompareEQ(),
>                               EvaluationStrategy.SEQUENTIAL, StorageStrategy.IN_MEMORY);
>       }
> +
> +     @Test
> +     public void test_isSorted() throws Exception {
> +             runTest(new SortingTests.TestIsSorted(),
> +                             EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
> +     }
> +
> +     @Test
> +     public void test_compareAndSwap() throws Exception {
> +             runTest(new SortingTests.TestCompareAndSwap(),
> +                             EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
> +     }
> +     @Test
> +     public void test_Sort() throws Exception {


I would probably create a new class for testing sorting in itself. We should at some point soon (when I get the time) create a test class for all arithmetic protocol suites that fulfil the requirements. But for now, just a separate class I think. Based on: Sorting is using comparison, but it more than that.

—
Reply to this email directly or view it on GitHub<https://github.com/aicis/fresco/pull/18/files#r54017987>.